README.md

```diff
@@ -1 +1,212 @@
-# JavaScript-Capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <img src="https://user-images.githubusercontent.com/78832703/203859927-696905b8-edee-4cc6-8642-0759f99b9ffd.png"
+alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 JavaScript Capstone Project ](#-javascript-capstone-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 JavaScript Capstone Project <a name="about-project"></a>
+
+> In this project, we are setting up a Ecommerce website with basic functionalities using JavaScript modules, Webpack and APIs.
+**My Shop** is a basic product listing page of an E-commerce website.
+The website display the list of products and allows users to like their favorite product, their can also chech the comments of others users about a specific product and finally the user can comment a product.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Building</summary>
+  <ul>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+    <li>HTML 5</li>
+    <li>CSS 3</li>
+    <li>ES6</li>
+  </ul>
+</details>
+<details>
+  <summary>Bundle</summary>
+  <ul>
+    <li><a href="https://webpack.js.org/">WebPack</a></li>
+    <li><a href="https://www.npmjs.com/">NPM</a></li>
+  </ul>
+  </details>
+<details>
+  <summary>Tests</summary>
+  <ul>
+    <li><a href="https://jestjs.io/">Jest</a></li>
+    <li><a href="https://babeljs.io/">Babel JS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://webpack.js.org/configuration/dev-server/">WebPack Dev Server</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://fakestoreapi.com/">Fakestore API</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Like the favorite Product**
+- **Check for comments of other users in a pop-up**
+- **Add new comment to the favorite products**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://vanelnw.github.io/JavaScript-Capstone/)
```

Comment:
- [ ] OPTIONAL
Live demo is pointing to the ReadMe. Kindly ensure to update your demo link accordingly.